assets/css/style.css

```diff
@@ -878,3 +729,84 @@ body {
     color: white;
   }
 }
+
+/* Memu barr */
+.topnav {
+  overflow: hidden;
+  background-color: #333;
+  padding-left: 90px;
+  padding-right: 90px;
+}
```

Comment:
Your design doesn't quite look like the template design.

Your design:

<img width="144" alt="Screen Shot 2022-06-20 at 10 23 39" src="https://user-images.githubusercontent.com/82426682/174634329-923c272b-9fc5-43d8-8451-424c6ab497f5.png">

The template design:

<img width="191" alt="Screen Shot 2022-06-20 at 10 24 15" src="https://user-images.githubusercontent.com/82426682/174634418-cfa5e831-2bc3-4666-a212-30ab831cc363.png">

Please fix it.